src/olympia/addons/templates/addons/impala/review_add_box.html

```diff
@@ -28,20 +26,15 @@ <h2>{{ _('What do you think?') }}</h2>
         {% endtrans %}
       </p>
       {% if addon.support_url or addon.support_email %}
-        <p>
-          See this add-on's 
-          {% if addon.support_url %}
-            <a class="support" href="{{ addon.support_url|external_url }}">{{ _('support section') }}</a>
-          {% elif addon.support_email %}
-            {{ emaillink(addon.support_email.localized_string, _('support email')) }}
-          {% endif %}
-          to find out if assistance is available.
+      <p>
+        {% trans support='#support' %}
+          See this add-on's <a href="{{ support }}">support section</a> to find out if assistance is available.
           You can also try asking the <a href="https://discourse.mozilla-community.org/c/add-ons/add-on-support">add-on community</a> for help.
-        </p>
+        {% endtrans %}
+      </p>
       {% endif %}
       <p><a href="{{ url('pages.review_guide') }}">{{ _('Review Guidelines') }}</a></p>
     </div>
 
   </div>{# /#review-box #}
 {% endif %}{# /is_author #}
-</div>
```

Comment:
Looks like one removed line too many, not sure if this happened during a merge wrongfully, needs checking.